docs/users-documentation.html

```diff
@@ -164,16 +152,37 @@
                         <ol>
                             <li>Clone the repository: git clone https://github.com/project-ncl/dependency-analysis</li>
                             <li>Enter the directory with the CLI: cd dependency-analysis/cli-wrap/src/main/resources/cli/</li>
-                            <li>Change permissions: chmod +x *.py *.sh</li>
-                            <li>Run the CLI. E.g.: ./da-cli.sh lookup org.hibernate:hibernate-core:4.2.21.Final</li>
+                            <li>Change permissions: chmod +x *.py</li>
+                            
                         </ol>
                 </p>
             </li>
+            
+            <li>
+                After unpacking/clonning CLI tool, you need to configure servers.
+            <p>
+                You have to specify DA server and KeyCloak authentication server. DA server can be set as environmental variable ($ export DA_SERVER=...).
+                Note that server must be inserted protocol-less because services are using different protocols.
+                Another option is inserting URL addresses od these servers to file config.json 
```

Comment:
/od/of/
But I would write it so that the config.json is understood as the main way to configure and the ENV variable is alternative.